lib/cucumber/formatter/ansicolor.rb

```diff
@@ -68,11 +68,10 @@ module ANSIColor
         'tag'       => 'cyan'
       )
 
-      if ENV['CUCUMBER_COLORS'] # Example: export CUCUMBER_COLORS="passed=red:failed=yellow"
-        ENV['CUCUMBER_COLORS'].split(':').each do |pair|
-          a = pair.split('=')
-          ALIASES[a[0]] = a[1]
-        end
+      # Example: export CUCUMBER_COLORS="passed=red:failed=yellow"
+      ENV['CUCUMBER_COLORS']&.split(':')&.each do |pair|
+        a = pair.split('=')
```

Comment:
```suggestion
        a = pair.split('=', 2)
```

Debating with myself: _Would it be better to split into 2 groups, only?_